jena-rdfconnection/src/main/java/org/apache/jena/rdflink/RDFLinkFuseki.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.jena.rdflink;
+
+import java.net.http.HttpClient;
+
+import org.apache.jena.rdflink.RDFLinkFuseki;
+import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.RDFFormat;
+import org.apache.jena.riot.resultset.ResultSetLang;
+import org.apache.jena.sparql.core.Transactional;
+
+/**
+ * Implementation of the {@link RDFLink} interface for connecting to an Apache Jena Fuseki.
+ * <p>
+ * This adds the ability to work with blank nodes across the network.
```

Comment:
👍 I think this is solving that JIRA issue where a user requested the ability to access blank nodes? Great job!